server/utilities.js

```diff
@@ -0,0 +1,97 @@
+var queries = require('./queries.js');
+
+module.exports.evalAlg = function(userInput, dataType) {
+	// TODO: add queries file, get data inputs from database, save in respective variables
+	var data = queries.getSets(dataType);
+
+	/** 
```

Comment:
Also, please convert tabs to spaces for all of your indentation. An indent should be two spaces.